### PR TITLE
Storing latest measurements on a kafka topic

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**

- [x] Stores latest measurements on a Kafka topic

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - []()
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: src/airflow/airqo_etl_utils/message_broker_utils.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            columns={"mongo_id": "device_id", "name": "device_name"}, inplace=True
+        )
+
+        del data["device_id"]
+
+        data = pd.merge(
+            left=data, right=devices, on=["device_number", "site_id"], how="left"
+        )
+        data["network"] = data["tenant"]
+        data["tenant"] = str(Tenant.AIRQO)
```
Comment (NoahNsimbe) [inline src/airflow/airqo_etl_utils/message_broker_utils.py:92]: adding network and tenant

Comment (Baalmart) [reply src/airflow/airqo_etl_utils/message_broker_utils.py:92]: Glad that tenant value is now fixed to AIRQO. Thanks @NoahNsimbe 

The one varying is the AirQo one.

Thanks.

**Review by Baalmart — APPROVED** [review]
Thanks @NoahNsimbe

File: src/airflow/airqo_etl_utils/message_broker_utils.py (hunk 1 above)
Comment (NoahNsimbe) [reply src/airflow/airqo_etl_utils/message_broker_utils.py:92]: The one varying is for the network. For example. KCCA network will be.
```
data["network"] = 'kcca'
data["tenant"] = str(Tenant.AIRQO)
```
@Baalmart

**Review by BenjaminSsempala — APPROVED** [review]
LGTM

**Comment by NoahNsimbe** [comment]
> A quick one @NoahNsimbe , what is the name of the "LATEST" measurements kafka topic? Is it a new topic? Just curious. cc: @BenjaminSsempala

`hourly-measurements-topic` @Baalmart

**Comment by Baalmart** [comment]
Thanks @NoahNsimbe

**Pull request merged**